app/helpers/deployment_helper.rb

```diff
@@ -2,4 +2,13 @@ module DeploymentHelper
   def deployment_count_options
     1.upto(6).to_a
   end
+
+  def provider_label(provider_name)
+    img_path = "providers/logo_#{provider_name.gsub(/\s/, '_').downcase}_small.png"
+    if File.exist?(Rails.root.join("app", "assets", "images", img_path))
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.